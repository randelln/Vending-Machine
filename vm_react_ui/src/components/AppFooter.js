import React, { Fragment } from 'react';

const AppFooter = () => {
  return (
    <>
      <hr className="featurette-divider" />
      <footer style={{ padding: '0 50px' }} className="navbar fixed-bottom">
        <p>Â© 2020 Randells Vending (PTY) LTD.</p>
      </footer>
    </>
  );
};

export default AppFooter;
