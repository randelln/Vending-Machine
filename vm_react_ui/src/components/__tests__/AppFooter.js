import React from 'react';
import { shallow } from 'enzyme';
import AppFooter from "../AppFooter";

describe('AppFooter', () => {
    const view = shallow(<AppFooter />);

    it('should render the AppFooter without issues', () => {
        expect(view).toMatchSnapshot();
    });
});
