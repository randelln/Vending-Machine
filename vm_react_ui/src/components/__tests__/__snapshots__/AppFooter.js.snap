// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter should render the AppFooter without issues 1`] = `
<Fragment>
  <hr
    className="featurette-divider"
  />
  <footer
    className="navbar fixed-bottom"
    style={
      Object {
        "padding": "0 50px",
      }
    }
  >
    <p>
      Â© 2020 Randells Vending (PTY) LTD.
    </p>
  </footer>
</Fragment>
`;
